windows-gaming-driver/src/config.rs

```diff
@@ -11,10 +11,16 @@ pub struct DeviceId {
     pub product: u16,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq, Copy)]
+pub struct DevicePort {
```

Comment:
No. As I said we use it for both USB and PCI. You could create one `UsbId` and one `PciId` struct but why bother when they do the same?